ui/src/components/addItemStep2.js

```diff
@@ -0,0 +1,178 @@
+import React, { useEffect, useRef, useState } from 'react';
+import DayPickerInput from 'react-day-picker/DayPickerInput';
+import {ONE_DAY_MILIS} from '../utils/constants';
+import "react-day-picker/lib/style.css";
+import "../styles/styles.css";
+
+function AddItemStep2(props){
+    let [disabledStart,setDisabledStart]=useState(new Date());
+    useEffect(()=>{
+        let ds=new Date();
+        ds.setHours(12,0,0,0)
+        setDisabledStart(ds);
+    },[]);
+
+    let [disabledEnd,setDisabledEnd]=useState(new Date());
```

Comment:
no need for state to be let

also, we should add structure to our file, one example would 
all useState's
all useEffects's
all methods
return value